docs/public/components/shared_features/flares.md

```diff
@@ -0,0 +1,119 @@
+# Flare
+
+The general idea is to register a callback within your component to be called each time a flare is created. This uses
+[Fx](../fx.md) groups under the hood, but helpers are there to abstract all the complexity.
+
+Then, migrate the code related to your component's domain from `pkg/flare` to your component and delete it from
```

Comment:
This reads like it's the last step in a series of steps. Did you read to write more before this?